File: C/Iniciante/1002 - Área do Círculo..c
/*
* Autor: Pierre Vieira
* Data de submissão: 13/09/2018 22:10:52
*/
#include <stdio.h>
#include <math.h>
int main()
{
    double area, raio;
    scanf("%lf",&raio);
    area=3.14159*pow(raio,2);
    printf("A=%.4lf\n",area);
}

File: Kotlin/Iniciante/1002 - Área do Círculo.kt
/*
* Autor: Pierre Vieira
* Data de submissão: 24/04/2021 16:07:30
*/
import java.lang.Math.pow
import java.math.BigDecimal
import java.math.RoundingMode

fun main(args: Array<String>) {

    val raio = readLine()!!.toDouble()
    val pi = 3.14159
    val area = pi * pow(raio, 2.0)
    val resultado = BigDecimal(area).setScale(4, RoundingMode.HALF_EVEN)
    println("A=$resultado")

}